packages/aws-client/src/services.ts

```diff
@@ -6,7 +7,7 @@ import * as AWS from 'aws-sdk';
 
 import awsClient from './client';
 
-export const apigateway = awsClient(AWS.APIGateway, '2015-07-09');
+export const apigateway = awsClient(ApiGatewayV2Client, '2015-07-09');
```

Comment:
Why do you use `ApiGatewayV2Client` instead of ``ApiGatewayClient`?  V2 has api version `2018-11-29`, the regular has api version `2015-07-09`

Reply:
Ah, I didn't recognize the difference at the time. I can update to use the V1 client

Reply:
https://github.com/nasa/cumulus/pull/2915/commits/f50ef09baecb175739c97cc4cec8f630cd96f7cd